### PR TITLE
Add description for system-contract-pub-key configuration parameter

### LINKED ISSUE #3748: Add description for system-contract-pub-key configuration parameter.
## Overview

Configuration parameter for public key which manages system contracts updates should have description in the default config file.

Description should be:
```
Public key to manage updates of system contracts
It should be set when network is started (genesis block is created). Not setting this key will disable possibility to update system contracts.
```

https://github.com/rchain/rchain/blob/b0376586b9b9e9f54c2e07e2ed6a748c19d23a77/node/src/main/resources/defaults.conf#L295

### PR DESCRIPTION
**Pull request by maximus19910112**
## Overview
issue #3748 

### Notes
Added description for system-contract-pub-key configuration parameter

Screenshot from 2022-07-02 18-25-35

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala [hunk 1]
```diff
@@ -455,7 +455,9 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
     )
 
     val systemContractPubKey = opt[String](
-      descr = "System contract public key"
+      descr = "Public key to manage updates of system contracts" +
+        "It should be set when network is started (genesis block is created)" +
+        "Not setting this key will disable possibility to update system contracts"
```
Comment (tgrospic) [inline node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala]: Can you please check and put in the description how this is printed in the terminal.

It seems there is no new line or period at the end of the sentence.

File: node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala [hunk 2]
```diff
@@ -455,7 +455,9 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
     )
 
     val systemContractPubKey = opt[String](
-      descr = "System contract public key"
+      descr = "Public key to manage updates of system contracts." +
+        "It should be set when network is started (genesis block is created)." +
+        "Not setting this key will disable possibility to update system contracts."
```
Comment (tgrospic) [inline node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala:460]: It would be nice to have space after period. :)

**Review by tgrospic — APPROVED** [review]
LGTM

**Review by zsluedem — APPROVED** [review]
LGTM

**Pull request merged**